src/slowmokit/methods/preprocessing/one_hot_encoder.hpp

```diff
@@ -0,0 +1,13 @@
+/**
+ * @file methods/preprocessing/one_hot_encoder.hpp
+ *
+ * Easy include To Return the one hot encoded data
+ */
+
+#ifndef ONE_HOT_ENCODER_HPP
+#define ONE_HOT_ENCODER_HPP
+#include "../../core.hpp"
+template<class T>
```

Comment:
Add documentation here i.e description of the parameters and its return value.
![Screenshot 2023-02-06 at 10 24 09 PM](https://user-images.githubusercontent.com/78130443/217034425-054d1d3f-1e69-42b2-9159-55f237e8e23a.png)

Reply:
done